WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/details/BackupDownloadDetailsViewHolder.kt

```diff
@@ -0,0 +1,32 @@
+package org.wordpress.android.ui.jetpack.backup.details
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.backup_details_list_item.view.*
+import org.wordpress.android.R
+import org.wordpress.android.ui.jetpack.backup.details.BackupDownloadDetailsViewModel.ListItemUiState
+import org.wordpress.android.ui.utils.UiHelpers
+
+sealed class BackupDownloadDetailsViewHolder(
+    internal val parent: ViewGroup,
```

Comment:
1) Question (❓ ): Why was the `internal` modifier used here?
2) Minor (🔍 ): The whole `internal val` can be removed, since it is just a past through. Wdyt?
3) Minor (🔍 ): Consider preserving the order in which the fields are defined, matching that of the parent. `layout` first and then `parent`.

Reply:
Changed in 3b2546f
Thanks for the tips